### DISCUSSION TITLE
Notes-based-on-Metarhia

### DISCUSSION THREAD

**Discussion by zziyad** [HowProgrammingWorks/LiveQA#29] [Code review]
It will be great if you review my [code](https://github.com/zziyad/Notes---based-on-Metarchia)

**Comment by tshemsedinov** [comment]
First issue: https://github.com/zziyad/Notes---based-on-Metarhia/commit/3a81d7618dccf4a01521bc6c301050b62c00ad65#r71513671
Please refactor this.

**Comment by tshemsedinov** [comment]
We have video recording of 1st review, see in telegram of your group

**Reply by zziyad** [reply]
Sorry for the late reply. I already made the changes. The link is below 
https://github.com/zziyad/Notes---based-on-Metarhia/commit/42ad69a1f65823ddca5d8cc4c7ff3c5c59f6f5b1
